wellcomeml/__main__.py

```diff
@@ -12,7 +12,7 @@ def download(download_target):
             'python', '-m', 'spacy', 'download', 'en_trf_bertbaseuncased_lg'])
     elif download_target == "non_pypi_packages":
         subprocess.run([
-            'pip', 'install', 'git+https://github.com/epfml/sent2vec.git'])
+            'build/virtualenv/bin/pip', 'install', 'git+https://github.com/epfml/sent2vec.git'])
```

Comment:
that assumes that there is a `build/virtualenv` in the user, a user might call their virtualenv differently, for example as i did in the demo `python3 -m venv venv`. i did the same mistake when first implemented it but eventually we decided with @aCampello that is better to delegate in which virtualenv to install to the user

Reply:
Oh yeah, that's true. `pip` would be preferable. I missed this from the review!

Reply:
hm good point, but when it was just pip install the tests failed (sent2vec not found). So it does seem to be neccessary to pip install it to a particular location

Reply:
(and these are the tests as run on Travis not my machine)

Reply:
is there a way we can do this systematically? i.e. know which `build/virtualenv/bin/pip` or equivalent location to install to?

Reply:
no, this is the responsibility of the user, this should not fail at the tests as the virtualenv should be enabled